providers/common/include/internal/ciphers/cipher_gcm.h

```diff
@@ -67,7 +67,7 @@ typedef struct prov_gcm_ctx_st {
     size_t bufsz;           /* Number of bytes in buf */
     uint64_t flags;
 
-    unsigned int pad : 1;   /* Whether padding should be used or not */
+    size_t pad;             /* Whether padding should be used or not */
```

Comment:
Why? The intention is obviously that other flags can be made into bit fields as well. Why do you want to waste a size_t on a single bit flag?